MANIFEST.in

```diff
@@ -5,6 +5,8 @@ include integration_requirements.txt
 include unittest_requirements.txt
 include pyproject.toml
 include requirements/requirements.common.txt
+include tools/scripts/l18n/post_translation.sh
+include tools/scripts/l18n/pre_translation.sh
```

Comment:
this is surprising .. we should not distribute these scripts as part of the resulting pypi package